website/views.py

```diff
@@ -204,83 +221,96 @@ def answer_comment(request):
+                for c in answer_comments:
+                    comment_creator = c.user()
+                    email = comment_creator.email
+                    comment_creator_emails.append(email)
+                    comment_creator_emails.append('forum-notifications@fossee.in')
+                # getting distinct uids
+                uids = set(uids)
+                uids.remove(request.user.id)
+                for uid in uids:
+                    notification = Notification()
+                    notification.uid = uid
+                    notification.pid = request.user.id
+                    notification.qid = answer.question.id
+                    notification.aid = answer.id
+                    notification.cid = comment.id
+                    notification.save()
+
+                    user = User.objects.get(id=uid)
+                sender_name = "FOSSEE Forums"
+                sender_email = "forums@fossee.in"
+                subject = "FOSSEE Forums - {0} - Comment has a reply"\
+                          .format(answer.question.category)
+                to = comment_creator_emails
+                url = settings.EMAIL_URL
+                message = """
+                    A reply has been posted on your comment.\n\n
+                    Title: {0}\n
+                    Category: {1}\n
+                    Link: {2}\n\n
+                    Regards,\nFOSSEE Team,\nIIT Bombay.
+                    """.format(
+                    answer.question.title,
+                    answer.question.category,
+                    # question.tutorial,
+                    'http://forums.fossee.in/question/' +
+                    str(answer.question.id) + "#answer" + str(answer.id)
+                )
+
+                send_mail(subject, message, sender_email, to)
+            except:
+                pass
             return HttpResponseRedirect("/question/" + str(answer.question.id))
-            print "------success in comment ---------"
+            # print "------success in comment ---------"
         else:
-            print "---------form not valid else part---------"
-            context.update({'form':form,
-                'question':answer.question,
-                'answers':answers})
-            return render(request, 'website/templates/get-question.html', context)
+            # print "---------form not valid else part---------"
+            context.update(
+                {'form': form,
+                 'question': answer.question,
+                 'answers': answers})
+            return render(request, 'website/templates/get-question.html',
+                          context)
     context.update(csrf(request))
-    context.update({'form':form,
-       'question':answer.question,
-       'answers':answers})
-    print "-----------end---------"
+    context.update(
+      {'form': form,
+       'question': answer.question,
+       'answers': answers})
+    # print "-----------end---------"
```

Comment:
Global warming is a problem. Print statements in views.py are a Bigger problem! Please remove this